File: CSharp/OpenTraceability/Models/MasterData/DigitalLink.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpenTraceability.Models.MasterData
{
    public class DigitalLink
    {
        public string link { get; set; } = string.Empty;
        public string title { get; set; } = string.Empty;
        public string linkType { get; set; } = string.Empty;
        public string ianaLanguage { get; set; } = string.Empty;
        public string context { get; set; } = string.Empty;
        public string mimeType { get; set; } = string.Empty;
        public bool active { get; set; }
        public bool fwqs { get; set; }
        public bool defaultLinkType { get; set; }
        public bool defaultIanaLanguage { get; set; }
        public bool defaultContext { get; set; }
        public bool defaultMimeType { get; set; }
        public string identifier { get; set; } = string.Empty;
        public bool authRequired { get; set; }
    }
}

File: Java/OpenTraceability/src/main/opentraceability/models/masterdata/digitallink.kt
package models.masterdata
import java.lang.reflect.Type
class DigitalLink {
    var link: String = ""
    var title: String = ""
    var linkType: String = ""
    var ianaLanguage: String = ""
    var context: String = ""
    var mimeType: String = ""
    var active: Boolean = Boolean()
    var fwqs: Boolean = Boolean()
    var defaultLinkType: Boolean = Boolean()
    var defaultIanaLanguage: Boolean = Boolean()
    var defaultContext: Boolean = Boolean()
    var defaultMimeType: Boolean = Boolean()
    var identifier: String = ""
    var authRequired: Boolean = Boolean()
    companion object{
    }
}
